elements/bulbs-video/components/revealed.js

```diff
@@ -274,6 +303,16 @@ export default class Revealed extends React.Component {
     }
   }
 
+  handlePauseEvent (reason) {
+    if (reason.pauseReason === 'external') {
+      return;
+    }
+    else if (reason.pauseReason === 'interaction') {
```

Comment:
Can we make the assumption that the absence of a `pauseReason` implies it is an ad pause or use some kind of player state?